source/docs/third-party-services/comments-and-widgets.md

```diff
@@ -238,3 +238,24 @@ rating:
 {% endcode %}
 <!-- endtab -->
 {% endtabs %}
+
+### Multiple Comment System Support
+
+NexT allows you to enable multiple commenting systems at the same time. You can modify the following configuration to adjust the style or behavior of multiple commenting systems.
+
+```yml
+# Multiple Comment System Support
+comments:
+  # Available values: tabs | buttons
+  style: tabs
+  # Choose a comment system to be displayed by default
+  # Available values: changyan | disqus | disqusjs | facebook_comments_plugin | gitalk | livere | valine | vkontakte
+  active:
```

Comment:
I don't quite understand how to enable two comment systems. the active value is just one systems?

Reply:
> I see, is it just works when two comment systems were enabled?

Yes